### PR TITLE
Add Avaje-Inject to DI docs

### PR DESCRIPTION
**Pull request by SentryMan**
As the title says

### CODE REVIEW

File: docs/asciidoc/di-avaje.adoc [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+----
+
+<1> Bootstrap avaje inject bean container
+<2> Use Avaje provided objects
+
+==== MVC routes
+
+Integration of MVC routes with Avaje is as simple as:
+
+.MVC and Dagger
```
Comment (jknack) [inline docs/asciidoc/di-avaje.adoc:98]: no dagger

**Review by jknack — COMMENTED** [review]
looks good, never tried it.

**Review by jknack — APPROVED** [review]

**Comment by jknack** [comment]
@SentryMan is it done?

**Comment by SentryMan** [comment]
Yes

**Pull request merged**